astro/src/content/quickstarts/quickstart-springboot-web.mdx

```diff
@@ -16,35 +16,17 @@ import LoginBefore from '../../diagrams/quickstarts/login-before.astro';
 import RemoteCode from '../../components/RemoteCode.astro';
 import SpringBootPreReqs from '../../components/quickstarts/springboot-prereqs.mdx';
 
-## Integrate Your Java Spring Application With FusionAuth
-This Quickstart will take you through building a simple Spring Boot web application and integrating it
-with FusionAuth. It uses Spring Boot web and Thymeleaf.
-You'll be building it for [ChangeBank](https://www.youtube.com/watch?v=CXDxNCzUspM), a global leader in converting dollars into coins.
-It'll have areas reserved for users who have logged in as well as public facing sections.
-
-In this Quickstart you'll create the Changebank application, which has
-* A Home Page, which is where a logged-out user goes. It has a login button that allows a user to log in
-using FusionAuth
-* An Account Page, which is where a user goes when they're logged in. It has a Logout button that allows
-the user to log out with FusionAuth
-* The associated back end routes to accomplish all of this
-
-This quickstart will be referencing code in the example application in the [Spring Boot Web Quickstart Repo](https://github.com/FusionAuth/fusionauth-quickstart-springboot-web).
-You can find a fully-working version of this application there.
-
-#### Repository Contents
-| Item | Description |
-| --- | --- |
-| /complete-application/ | A completed application, with a working integration with FusionAuth |
-| /kickstart/ | A directory containing a `kickstart` file, which is used to configure FusionAuth |
-| /stub-application/ | A downloaded and unzipped but otherwise clean start to the Spring Boot template with dependencies |
-| /.env | Environment variables used by the dokcer-compose file |
-| /README.md | Readme file for the repo |
-| /docker-compose.yml | A docker-compose config file for standing up a FusionAuth server, a Postgres database, and an Elastic instance |
+
+
+In this quickstart, you will build a Spring Boot web application and integrate it with FusionAuth. The application is for [ChangeBank](https://www.youtube.com/watch?v=pkH-kD73QUM), a global leader in converting dollars into coins. It will have areas reserved for logged in users and public-facing sections. It uses Spring Boot web and Thymeleaf, and sets up an OAuth authorization code grant flow, allowing users to log in to the application.
+
+Find the Docker Compose file and source code for the complete application at https://github.com/FusionAuth/fusionauth-quickstart-springboot-web.
+
+## Prerequisites
 
 <SpringBootPreReqs/>
```

Comment:
Should there be a minimum version or at least a 'this quickstart was written with this version of Java'

Reply:
I see there is a note at the end of the prereqs about Java 17.  That should be moved up to the Java section.